docs/product/integrations/index.mdx

```diff
@@ -53,6 +54,7 @@ For more details, see the [full Integration Platform documentation](/product/int
 | [Height](/product/integrations/issue-tracking/height/)                            | X                      | X                        | X          |
 | [Incident.io](/product/integrations/issue-tracking/incidentio/)                   | X                      |                          | X          |
 | [Jira/Jira Server](/product/integrations/issue-tracking/jira/)                    | X                      | X\*                      | X          |
+| [Kitemaker](/product/integrations/issue-tracking/kitemaker/)                      | X                      |                          | X          |
```

Comment:
This link isn't showing up because the page is actually in the "notification-incidents" not the "issue-tracking" folder :)